Automattic/jetpack
# AI Assistant: Add "Create a form" toolbar action

Fixes #32521

## Proposed changes:

* Adds a second `MenuGroup` with an option to create a form, that replaces the assistant with a new empty form

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Go to the block editor
* Add an AI Assistant block
* Click on `Write with AI...` > `Create a form...`
* Check that the assistant is replaced with a form
* Check that there is no regression

![2023-10-10_16-04-35](https://github.com/Automattic/jetpack/assets/8486249/6970aeac-0397-4145-8dcb-f0df12fe8b62)
